src/components/Header/Header.jsx

```diff
@@ -134,6 +134,8 @@ export function Header(props) {
   const canAccessPopups =
     props.hasPermission('createPopup', !isAuthUser && canInteractWithViewingUser) ||
     props.hasPermission('updatePopup', !isAuthUser && canInteractWithViewingUser);
+  // SendEmails
+  const canAccessSendEmails = props.hasPermission('sendEmails', !isAuthUser);
```

Comment:
```suggestion
  const canAccessSendEmails = props.hasPermission('sendEmails');
```
hasPermission only accepts one argument, right?